SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -8,17 +8,28 @@
 %define config_source %{SOURCE1}
 %endif
 
+%ifarch aarch64
+%global __provides_exclude_from %{_libdir}/debug/.build-id/
+%define arch arm64
+%define archdir arm64
+%define config_source %{SOURCE2}
+%endif
+
+# Generated by update script
+%global kernelver 5.15.98.mshv1
+
 Summary:        Linux Kernel for Kata UVM
 Name:           kernel-uvm
-Version:        5.15.48.1
-Release:        9%{?dist}
+Version:        %{kernelver}
+Release:        1%{?dist}
 License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Kernel
 URL:            https://github.com/microsoft/CBL-Mariner-Linux-Kernel
-Source0:        https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-2/%{version}.tar.gz#/kernel-%{version}.tar.gz
+Source0:       %{_mariner_sources_url}/kernel-mshv-%{kernelver}.tar.gz
 Source1:        config
+Source2:        config
```

Comment:
Potentially we also want to make sure this config doesn't get changed by anybody. This is a copy of the config in LSG dom0 sources, so whatever kernel dom0 this currently builds on, this file will only be a copy (immutable by hand). We could add a comment for this